prov/gni/include/gnix_mr.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2015 Cray Inc. All rights reserved.
+ *
```

Comment:
@hppritcha Do you know if the license must be included in every file?  I don't see that on the libfabric wiki, but regardless, it seems like a good idea.